crates/ruma-client-api/src/error.rs

```diff
@@ -192,14 +197,24 @@ pub enum ErrorKind {
     _Custom { errcode: PrivOwnedStr, extra: Extra },
 }
 
+impl ErrorKind {
+    /// Constructs an empty [`ErrorKind::Forbidden`] variant.
+    pub fn forbidden() -> Self {
+        Self::Forbidden {
+            #[cfg(feature = "unstable-msc2967")]
+            authenticate: None,
+        }
+    }
```

Comment:
Do we need another c'tor to conveniently create `Forbidden` w/ `authenticate: Some(_)`?

Reply:
Yes, that could be helpful. Added.